clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -380,6 +381,30 @@ public void testNetworkTimeout() {
         assertEquals(1, result.unsentRequests.size());
     }
 
+    @Test
+    public void testDisconnect() {
```

Comment:
How about `testDisconnectAndRetry` ?

Reply:
I think retry is what we truly expect, so we might not need to specify it redundantly.